analyzer/optane/check_fw_error_count

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+#set -x
+
+# Prevent repeated sourcing
+if [ -n "${__MODULE_CHECK_FW_ERROR_COUNT__}" ]; then
+  return
+fi
+__MODULE_CHECK_FW_ERROR_COUNT__="Loaded"
+
+# check percentage remaining
+function optane_check_FW_Error_Count() {
+  local FNAME=$1        # File name to process
+  local ERR_STATE=false # Used for error reporting
+
+  # Exit the function with INFO message if the file to process doesn't exist
+  if [[ ! -f ${FNAME} ]] ; then
+    info_msg " Could not process '${FNAME}'. File not found"
+    REPORT_COUNT_INFO=$((REPORT_COUNT_INFO+1)) # Increment the INFO count
+    return
+  fi
+
+  # Get the DimmID and Current Value(FWError Count) fields from the input file
+  while IFS='|' read -r DimmID Type CurrentValue
+  do
+    DimmID="$(echo ${DimmID})"
+    CurrentValue=(${CurrentValue})
+
+    # Check the FWErro Count
+
+    if [[ ${CurrentValue} -lt 5 ]] && [[ ${CurrentValue} -ge 2 ]]
```

Comment:
Perhaps we should increase the ranges as some errors are acceptable
10-49 = INFO
50-100 = WARN
>100 = CRIT